src/log_capture.jl

```diff
@@ -108,6 +114,11 @@ _has_logs(ts::TestSetup) = filesize(logpath(ts)) > 0
 # The path might not exist if a testsetup always throws an error and we don't get to actually
 # evaluate the test item.
 _has_logs(ti::TestItem, i=nothing) = (path = logpath(ti, i); (isfile(path) && filesize(path) > 0))
+_mem_watermark() = string(
+    "maxrss ", lpad(Base.Ryu.writefixed(100Float64(Sys.maxrss()/Sys.total_memory()), 1), 4),
+    "% | mem ", lpad(Base.Ryu.writefixed(100Float64(1 - (Sys.free_memory()/Sys.total_memory())), 1), 4),
+    "% | "
```

Comment:
i wonder if we could print this even more succinctly

it's nice that it's on the left of the RUNNING/FINISHED statement because (i) it keeps things fixed width, so logs line up nicely and (ii) it is more a property of the machine at that time that the specific testitem

but, it does mean the `item "foo"` statement is way to the right of the screen
```
Worker 2983:  02:39:53 | maxrss  7.4% | mem 16.4% | RUNNING  (  1/154) item "foo"
Worker 2983:  02:40:48 | maxrss  7.4% | mem 21.4% | FINISHED (  1/154) item "foo"
```

I wonder if we at least put them in the same column?

```
Worker 2983:  02:39:53 | maxrss  7.4% mem 16.4% | RUNNING  (  1/154) item "foo"
Worker 2983:  02:40:48 | maxrss  7.4% mem 21.4% | FINISHED (  1/154) item "foo"
```

Reply:
sounds good to me